hack/remove-members.go

```diff
@@ -0,0 +1,179 @@
+package main
+
+import (
+	"bufio"
+	"flag"
```

Comment:
Can we use https://github.com/spf13/pflag instead of flag? This will enable us to have POSIX style flags. (`--foo`)